packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.js

```diff
@@ -66,10 +67,9 @@ export default ({ coreSagas, networks }) => {
         payment = yield payment.from(defaultIndex, ADDRESS_TYPES.ACCOUNT)
         if (to) payment = yield payment.to(to)
       }
-      const defaultFeePerByte = path(
-        ['fees', feeType || 'regular'],
-        payment.value()
-      )
+      const defaultFeePerByte = payPro
+        ? (payPro.feePerByte + 1) | 0
```

Comment:
Not sure I'm following this logic. in what situation would this bitwise ever result in the "or" condition (i.e 0)? 

(`null`, `undefined`) + 1 will be 1 and therefor the bitwise seems unnecessary.  is this intended?

Reply:
Logic is, if new saga has payPro information, then set defaultFeePerByte to (paypro + 1) | 0, (the |0 truncates the decimal places (clean way to do it),  so an int value can be passed in as feePerByte), if new send saga does not have payPro then go forward with regular feePerByte setting